deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
Won't that cause the default setting to be used? For a build with system llvm, `LLVM_VER` is expected to not be set by the user and the version from `llvm-config` should be used instead of the default value.